testutil/testclient/testblock/client.go

```diff
@@ -104,6 +104,7 @@ func NewAnyTimeLastNBlocksBlockClient(
 	// returns the mock block.
 	blockClientMock := mockclient.NewMockBlockClient(ctrl)
 	blockClientMock.EXPECT().LastNBlocks(gomock.Any(), gomock.Any()).Return([]client.Block{blockMock}).AnyTimes()
```

Comment:
Can we remove this given that we added the one below or do we need both?

Reply:
We are not using `LastNBlocks` anywhere, I think it's safe to remove.